R/theme-elements.r

```diff
@@ -120,6 +121,37 @@ element_text <- function(family = NULL, face = NULL, colour = NULL,
   )
 }
 
+#' @param colour.accent1,color.accent1 accent colour 1,
+#'   typically a lighter version of colour
+#' @param colour.accent2,color.accent2 accent colour 2,
+#'   typically a bright colour used for geom_smooth et al.
+#' @param fill.accent accent fill colour, typically a darker version of fill
+#' @param alpha colour/fill transparency, between 0 & 1.
+#' @export
+#' @rdname element
+element_geom <- function(fill = NULL, fill.accent = NULL,
+                         colour = NULL, color = NULL,
+                         colour.accent1 = NULL, color.accent1 = NULL,
```

Comment:
What if we used `fill` and `fill_1`?

And what if we ditched `colour` + `color` in favo(u)r of `col`?

Reply:
I.e., take the argument names of `element_geom()` and run through this function:
https://github.com/tidyverse/ggplot2/blob/4d2ca992dc37fee598d83d3d3f524dbba38f4187/R/aes.r#L155

Reply:
However, this would require a `...` argument instead of listing everything explicitly, so maybe not the right approach. In any case, I’d argue against `col` just because in the rest of ggplot2 `col` is not used. And I think we also should talk about whether it’s `colour.accent1` or `colour_accent1`. We’re not entirely consistent with points vs underscores, and it would be good to pick a scheme and at least be consistent going forward.